platform-ui-skin/src/main/webapp/skin/less/social/portlets/profileSettings/Style.less

```diff
@@ -0,0 +1,104 @@
+/*
+This file is part of the Meeds project (https://meeds.io/).
+Copyright (C) 2022 Meeds Association
+contact@meeds.io
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU Lesser General Public
+License as published by the Free Software Foundation; either
+version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+Lesser General Public License for more details.
+You should have received a copy of the GNU Lesser General Public License
+along with this program; if not, write to the Free Software Foundation,
+Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+*/
+
+@import "../../../variables.less";
+@import "../../../mixins.less";
+#profileSettings {
+    background: @baseBackground!important;
```

Comment:
It would be better IMO to not increase the CSS content. If a portlet like `General Settings` doesn't [have a CSS](https://github.com/Meeds-io/platform-ui/blob/develop/platform-ui-skin/src/main/webapp/skin/less/social/portlets/GeneralSettings/Style.less), only imported the The cropper Component CSS, I think for this Portlet can make it using re-usable components and CSS helpers without defining custom styles at all. Could you please re-consider the definition here to avoid increasing the imported CSS in pages.

Reply:
We tried to not add specific style, but to respect the design provided it was necessary.

Reply:
ok, I specified more some examples to improve, by reusing existing helpers classes and avoid defining specific classes for applications to improve evolutivity, maintainability and pages rendering performances.